torch/csrc/jit/passes/quantization.cpp

```diff
@@ -30,9 +30,20 @@ int groups, bool benchmark, bool deterministic, bool cudnn_enabled) -> Tensor"};
   return quantnodeLookup.find(n);
 }
 
+void travAndReplaceQuantnodeinput(Node* dq, Value* v_to_quant) {
```

Comment:
Best practices are for a function to do only one thing - is there a reason to include `replaceInputWith` call here? That's the only reason we have the second argument here and it's completely unnecessary. Basically, we had
```
doFirstThing(a);
doSecondThing(b);
```
And this changed it to `doFirstAndSecondThing(a, b)`.

Sorry if I was not completely clear in my initial comment, but the idea was to only factor out
`dq->inputs()[0]->node()->inputs()[0]->node()`.

Reply:
i will make this change all together once i hear for your response on below items

Reply:
Done